framework.py

```diff
@@ -57,6 +57,8 @@ def launch_desktop_server(self, host, user_id):
 
         :param str host: Host for which we desire to answer requests.
         :param int user_id: Id of the user for which we desire to answer requests.
+        :param parent: Parent widget for any pop-ups to show during initialization.
```

Comment:
No, because the dialogs are short lived and not permanent. The `parent` is necessary so the dialogs always appear on top of the Shotgun Desktop, otherwise having the desktop in "On Top" mode would make the popup appear behind the window.